pkg/apis/networkaddonsoperator/shared/networkaddonsconfig_types.go

```diff
@@ -81,7 +81,11 @@ type KubeMacPool struct {
 }
 
 // MacvtapCni plugin allows users to define Kubernetes networks on top of existing host interfaces
-type MacvtapCni struct{}
+type MacvtapCni struct {
+	// DevicePluginConfig allows the user to override the name of the
+	// `ConfigMap` where the device plugin configuration is held.
```

Comment:
do we need to add this `// +optional` (like in [here](https://github.com/kubevirt/kubevirt/blob/main/staging/src/kubevirt.io/api/core/v1/componentconfig.go#L19))?

Reply:
why would we ? I see the KubeMAcPool / SecondaryDNS configurations do not have it.

Reply:
Was just think aloud, good point.